src/odemis/gui/util/img.py

```diff
@@ -1876,27 +1874,22 @@ def get_ordered_images(streams, raw=False):
             if isinstance(data, tuple): # 2D tuple = tiles
                 data = img.mergeTiles(data)
         else:
-            if isinstance(s.raw, tuple): # 2D tuple = tiles
+            if isinstance(s, RGBProjection):
+                data_raw = s.projectAsRaw()
+                if isinstance(data_raw, tuple):
```

Comment:
In such case, `.projectAsRaw()` should do the "right thing", which apparently is to do `img.mergeTiles(s.raw)`.